aspnetcore/fundamentals/index.md

```diff
@@ -105,13 +105,13 @@ For more information, see [Web Host](xref:fundamentals/host/web-host).
 
 In ASP.NET Core 3.0, Web Host (`WebHost` class) or Generic Host (`Host` class) can be used in a web app. Generic Host is recommended, and Web Host is available for backwards compatibility.
 
-The framework provides `CreateDefaultBuilder` and `ConfigureWebHostDefaults` extension methods that set up a host with commonly used options, such as the following:
+The framework provides `CreateDefaultBuilder` convenience and `ConfigureWebHostDefaults` extension methods that set up a host with commonly used options, such as the following:
```

Comment:
The framework provides the `CreateDefaultBuilder` and `ConfigureWebHostDefaults` methods that set up a host with commonly used options, such as the following: